pom.xml

```diff
@@ -98,7 +96,7 @@
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
-                <version>5.11.4</version>
+                <version>5.12.0</version>
```

Comment:
We manage both `junit-bom` and `hamcrest` in the plugin parent POM, so likely this section and the following one are unnecessary.

Reply:
Thanks!  Removed in c368a4ad8a9a334fab37cab0cc1049950f2c6c0d